src/PlatformHelper.cs

```diff
@@ -450,6 +450,17 @@ internal static DateTimeOffset ConvertStringToDateTimeOffset(string text)
         }
 
         /// <summary>
+        /// Converts a string to a DateTimeOffset.
+        /// </summary>
+        /// <param name="text">String to be converted.</param>
+        /// <returns>See documentation for method being accessed in the body of the method.</returns>
+        internal static DateTime ConvertStringToDateTime(string text)
+        {
+            var offset = ConvertStringToDateTimeOffset(text);
+            return offset.UtcDateTime;

```

Comment:
In your scenario, what's the format of the text? I suppose it is like '2016-10-25T17:49:27.6331167' which missed the TimeZone info. Then it will throw exception when validating the time zone.